### PR TITLE
Add files required for lgtm.vmware.run

### PR DESCRIPTION
**Pull request by caglar10ur**
\cc @frapposelli

### CODE REVIEW

File: .lgtm [hunk 1]
```diff
@@ -0,0 +1,2 @@
+approvals = 1
+pattern = "(?i):shipit:|:\\+1:|LGTM|:sheep :it"
```
Comment (frapposelli) [inline .lgtm]: shouldn't this be `pattern = "(?i):shipit:|:\\+1:|LGTM|:sheep: :it:"` ?

Comment (caglar10ur) [inline .lgtm]: Ah, done!

**Comment by dougm** [comment]
lgtm

**Comment by caglar10ur** [comment]
we still need @frapposelli to enable this :)

**Pull request merged**